variables.sh

```diff
@@ -24,6 +24,7 @@ FT_MC_CO_SERVER_LABEL=${FT_MC_CO_SERVER_LABEL:-submariner.io/gateway=true}
 NET_ATTACH_DEF_NAME=${NET_ATTACH_DEF_NAME:-ftnetattach}
 SRIOV_RESOURCE_NAME=${SRIOV_RESOURCE_NAME:-openshift.io/mlnx_bf}
 TEST_IMAGE=${TEST_IMAGE:-quay.io/wizhao/ft-base-image:0.9}
+IPERF3_SERVER_NAME="iperf3-server"
```

Comment:
We can be more specific here E.g.
`EXTERNAL_IPERF_SERVER_POD_NAME="external-iperf-server"`

Reply:
Can we refactor IPERF3_SERVER_NAME -> EXTERNAL_IPERF_SERVER_POD_NAME ?